### PR TITLE
@xtina-starr => [Onboarding] Passes forceStep to Wizard from slug

### PR DESCRIPTION
**Pull request by kanaabe**
~~Need to wait on https://github.com/artsy/reaction/pull/511~~

### CODE REVIEW

File: desktop/apps/personalize/components/App.js [hunk 1]
```diff
@@ -9,13 +9,19 @@ import Budget from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Budge
 
 export class App extends React.Component {
   static propTypes = {
-    currentUser: PropTypes.object
+    currentUser: PropTypes.object,
+    redirectTo: PropTypes.string,
+    forceStep: PropTypes.string
```
Comment (xtina-starr) [inline desktop/apps/personalize/components/App.js]: @kanaabe what's the purpose of these static propTypes again?

**Comment by xtina-starr** [comment]
This looks good to me. Just had a couple of questions. Merge whenever!

File: desktop/apps/personalize/components/App.js (hunk 1 above)
Comment (kanaabe) [reply desktop/apps/personalize/components/App.js]: So our linters are happy! Haha just kidding. It's so we can define types on our props. Similar to TS interface/type definitions but we don't have TS in Force, yet..

File: desktop/apps/personalize/index.coffee [hunk 2]
```diff
@@ -10,4 +10,5 @@ app = module.exports = express()
 app.set 'views', __dirname
 app.set 'view engine', 'jade'
 
-app.get '/personalize*', index
+app.get '/personalize', index
+app.get '/personalize/:slug', index
```
[the remark this answers is not in the document]
Comment (kanaabe) [reply desktop/apps/personalize/index.coffee]: Ah it's so we can extract the slug with `req.params.slug`

File: desktop/apps/personalize/components/App.js (hunk 1 above)
Comment (yuki24) [reply desktop/apps/personalize/components/App.js]: W have an open PR #2145 once we merge it we’ll be able to use TS, but this is :+1: for now

**Comment by xtina-starr** [comment]
👍

**Pull request merged**